repose-aggregator/core/repose-core-api/src/main/resources/META-INF/schema/system-model/system-model.xsd

```diff
@@ -106,6 +106,28 @@
                     </xs:annotation>
                 </xs:attribute>
 
+                <xs:attribute name="chunked-encoding" type="mod:ChunkedEncoding" use="optional" default="true">
```

Comment:
I'm sure we talked about it, but why did we move this here again?

Reply:
Chunked encoding, as we call it, is a property of a request, not a client. Therefore, I thought it should be configured for the requestor, not the client.

Configuring it for the client meant that we were setting a parameter on the client itself. To use that parameter, we would get the client when we were about to route a request to the origin service and retrieve the parameter by name. If you take a look at the old code, you can see how indirect that is. Also, the client API deprecated parameters on the client.